elevenlabs/api/voice.py

```diff
@@ -133,10 +133,35 @@ def from_design(cls, voice_design: VoiceDesign):
     def computed_settings(cls, v: VoiceSettings, values) -> VoiceSettings:
         url = f"{api_base_url_v1}/voices/{values['voice_id']}/settings"
         return v if v else VoiceSettings(**API.get(url).json())
+    
+    @classmethod
+    def default_settings(cls):
+        url = f"{api_base_url_v1}/voices/settings/default"
+        return VoiceSettings(**API.get(url).json())
 
     def delete(self):
         API.delete(f"{api_base_url_v1}/voices/{self.voice_id}")
 
+    # def edit_settings(self, stability: float, similarity_boost: float):
+    def edit_settings(self, voice_settings: VoiceSettings):
+        url = f"{api_base_url_v1}/voices/{self.voice_id}/settings/edit"
+        json = {
+            "stability": voice_settings.stability,
+            "similarity_boost": voice_settings.similarity_boost,
+        }
+        API.post(url, json=json)
+    
+    def edit(
+        cls,
```

Comment:
If you use `cls` you must provide the id as input `def edit(cls, voice_id,...)` if you want to use the id form the self object then you have to use `self` instead